spec/features/users/unauth_user_cant_see_private_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe "Unauthenticated user" do
+  scenario "user can't see others' private data" do
+
+   visit orders_path
+
```

Comment:
This is really interesting to me. does this visit orders without a current_user? Pretty elegant.